src/lib/PostLayout.svelte

```diff
@@ -35,6 +39,9 @@
   {#if updated}
     <meta itemprop="dateModified" content={updated} />
   {/if}
+  {#if thumbnail}
+    <meta itemprop="thumbnailUrl" content="https://96d89e5aa2f6325be95a9b0a29419d9d.r2.cloudflarestorage.com/vospel-cz/posts/{encodeURIComponent(thumbnail)}" />
```

Comment:
Move content to separate computes property `thumbnailUrl`.

https://pub-dc4c4b46dcdb4b8cafe7c4122362233b.r2.dev should be used